extensions/data-workspace/src/services/workspaceService.ts

```diff
@@ -26,23 +26,20 @@ export class WorkspaceService implements IWorkspaceService {
 	private openedProjects: vscode.Uri[] | undefined = undefined;
 	private excludedProjects: string[] | undefined;
 
-	constructor() {
-		this.getProjectsInWorkspace(undefined, true).catch(err => console.error('Error initializing projects in workspace ', err));
-
-		TelemetryReporter.createActionEvent(TelemetryViews.WorkspaceTreePane, TelemetryActions.ProjectsLoaded)
```

Comment:
Removed this call to `getProjectsInWorkspace` because it was unnecessary. When the projects viewlet is opened, `getProjectsInWorkspace` gets called by the tree loading done in `WorkspaceTreeDataProvider.getChildren()`

Also removing this telemetry here since it wasn't sending the open and excluded project count correctly because `getProjectsInWorkspace` wasn't being awaited.

Reply:
nevermind had to add this back. This was added in https://github.com/microsoft/azuredatastudio/pull/16650 to fix a bug where projects weren't loaded correctly when creating a new project from db and the projects viewlet hadn't been opened yet.